vscode/src/completions/providers/create-provider.ts

```diff
@@ -150,6 +154,51 @@ export async function createProviderConfig(
     return createAnthropicProviderConfig({ client })
 }
 
+async function resolveFinetunedModelProviderFromFeatureFlags(): Promise<{
+    provider: string
+    model?: FireworksOptions['model'] | AnthropicOptions['model']
+} | null> {
```

Comment:
To keep return types in sync automatically:

```suggestion
async function resolveFinetunedModelProviderFromFeatureFlags(): ReturnType<
    typeof resolveDefaultProviderFromVSCodeConfigOrFeatureFlags
> {
```

Reply:
magic